aioseop_class.php

```diff
@@ -4717,7 +4717,8 @@ function admin_menu() {
 			$menu_name, $menu_name, apply_filters( 'manage_aiosp', 'aiosp_manage_seo' ), $file, array(
 				$this,
 				'display_settings_page',
-			)
+			),
```

Comment:
Since you're modifying this call function, could you change it to...

```
add_menu_page(
	$menu_name,
	$menu_name,
	apply_filters( 'manage_aiosp', 'aiosp_manage_seo' ),
	$file,
	array( $this, 'display_settings_page' ),
	aioseop_get_menu_icon()
);
```

It'll make it easier to read. I was trying to double check the function params, and I think you mentioned using the wrong line at one point; not sure if this is the same set of lines.

Reply:
Same set of lines. I was calling aioseop_get_menu_icon() within the array above it which is why it wasn't working.